index.js

```diff
@@ -116,12 +116,93 @@ LocationForecast.prototype = {
       if (err) {
         return callback(err, null);
       }
-      
+
       return callback(null, res);
     });
   },
 
 
+    /**
+     * Get seven day weather.
+     * @param {Function} callback
+     */
+    getSevenDaySummary: function(callback) {
```

Comment:
Awesome, would be good if you added it here too. Yeah, in your package you can add:

```
"yr.no-forecast": "slandersson/yr.no-forecast#BRANCH_OR_TAG_NAME"
```